### DISCUSSION TITLE
Uuden sanan lisääminen: "sidostamaton"

### DISCUSSION THREAD

**Discussion by tlikonen** [voikko/discussion#7] [General]
Haluaisin lisätä Voikkoon sanan "sidostamaton". Osaan tietenkin lisätä Joukahaiseen sanoja, mutta kun tuo sana on johdos verbistä, varmistan ensin täältä. Kannattaako mainittu sana lisätä sellaisenaan, vai saako sana mukaan jonkin verbin johdoksena?

"Sidostamatonta" käytetään kuvailemaan tekstiä, jossa asioita ei ole kytketty toisiinsa kovin hyvin eli teksti tuntuu irrallisten asioiden luettelemiselta. Sen vastakohta on sidosteinen teksti.

**Comment by tlikonen** [comment]
Ei vastauksia, vielä. Lisäsin sanan "sidostamaton". Muutetaan tätä tietuetta (ks. linkki alla) myöhemmin, jos sana myöhemmin tunnistuu jotakin muuta kautta.

https://joukahainen.puimula.org/word/edit?wid=539734

**Comment by ahomansikka** [comment]
Sidostamaton kaiketi tunnistetaan, jos sidostaa lisätään Joukahaiseen.

Vertaa puhua -> puhuma -> puhumaton

**Comment by tlikonen** [comment]
* 2023-09-29 01:16:13-0700, ahomansikka wrote:

> Sidostamaton kaiketi tunnistetaan, jos sidostaa lisätään Joukahaiseen.
>
> Vertaa puhua -> puhuma -> puhumaton

Lisäsin jo "sidostamattoman" mutta pitäisikö sittenkin poistaa ja
korvata sidostaa-verbillä ja sen automaattisilla (?) johdoksilla?

-- 
/// Teemu Likonen - .-.. https://www.iki.fi/tlikonen/
// OpenPGP: 6965F03973F0D4CA22B9410F0F2CAE0E07608462

**Comment by ahomansikka** [comment]
Kannatetaan, ja samalla voisi lisätä laatusanan sidosteinen.

**Comment by tlikonen** [comment]
* 2023-09-29 23:46:59-0700, ahomansikka wrote:
> Kannatetaan, ja samalla voisi lisätä laatusanan sidosteinen.

Nyt on uusi verbi "sidostaa":
https://joukahainen.puimula.org/word/edit?wid=539736

"Sidostamaton" varustettu lipulla "siirretty":
https://joukahainen.puimula.org/word/edit?wid=539734

Adjektiivi "sidosteinen" on ollut olemassa jo suunnilleen aikojen alusta
saakka:
https://joukahainen.puimula.org/word/edit?wid=513863
